docs/docs/04-standard-library/06-math/api-reference.md

```diff
@@ -0,0 +1,772 @@
+---
+title: API reference
```

Comment:
I noticed something went wrong here. We have both 06-ex and 06-math, while we already have 07-math on the doc site: https://www.winglang.io/docs/standard-library/math/api-reference (which is defined in https://github.com/winglang/wing/blob/main/docs/docs/04-standard-library/07-math)

Reply:
fixed the numbers